backend/clickhouse/logs_test.go

```diff
@@ -964,6 +964,27 @@ func TestReadLogsWithServiceVersionFilter(t *testing.T) {
 	assert.Len(t, payload.Edges, 2)
 }
 
+func TestReadLogsWithMultipleFilters(t *testing.T) {
+	ctx := context.Background()
+	client, teardown := setupTest(t)
+	defer teardown(t)
+
+	now := time.Now()
+	rows := []*LogRow{
+		NewLogRow(now, 1,
+			WithLogAttributes(map[string]string{"code.lineno": "162", "os.type": "linux"})),
```

Comment:
Should we have a row that does not fall into this filter, but only one of them?

Reply:
Good idea. Added in 965ddf2d7